packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -230,6 +231,28 @@ describe("Garbage Collection Tests", () => {
 				assert(gc.gcEnabled, "gcEnabled incorrect");
 				assert.equal(gc.latestSummaryGCVersion, 1, "latestSummaryGCVersion incorrect");
 			});
+			it("gcEnforcementMinCreateContainerRuntimeVersion later than persisted value, sweepEnabled false", () => {
+				gc = createGcWithPrivateMembers(
+					{ createContainerRuntimeVersion: "1.2.3", sweepEnabled: true },
+					{ gcEnforcementMinCreateContainerRuntimeVersion: "2.0.0" },
```

Comment:
So the biggest problem with using string values here is either you have to write your own semver comparison algorithm, which is going to be challenging, or you're going to have to import a semver comparison algorithm which will cause the bundle size to increase.